src/resolvers-cassandra/Settings/index.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
   removeKeywords: mutations.removeKeywords,
   addKeywords: mutations.addKeywords,
 
-  terms: queries.terms,
+  siteTerms: queries.siteTerms,
```

Comment:
This might be more clear if it were named `siteTopics` so that it is consistent with `cassandra-setup.cql`.